docs/_interface/devp2p.md

```diff
@@ -0,0 +1,182 @@
+---
+title: devp2p
+sort-key: Q
+---
+
+[DevP2P](https://github.com/ethereum/devp2p) is a set of network protocols 
+that form the Ethereum peer-to-peer network. The DevP2P specifications 
+define precisely how nodes should find each other and communicate. 
+Geth implements the DevP2P specifications in Go.
+
+The DevP2P stack includes the low-level peer-to-peer protocols that 
+define discovery and secure sessions between nodes such as:
+
+- [Ethereum Node Records](https://github.com/ethereum/devp2p/blob/master/enr.md): A standard format for connectivity information for a node
+- [Discovery protocol](https://github.com/ethereum/devp2p/blob/master/discv4.md): Defines how nodes find each other.
+- [RLPx protocol](https://github.com/ethereum/devp2p/blob/master/rlpx.md): Defines a TCP based transport system for communication between nodes.
+
+DevP2P also includes the RLPx-based application level protocols including:
+
+- [Ethereum Wire Protocol](https://github.com/ethereum/devp2p/blob/master/caps/eth.md): facilitates exchange of blockchain data between peers
+- [Ethereum Snapshot Protocol](https://github.com/ethereum/devp2p/blob/master/caps/snap.md): enables exchange of snapshots between peers
+- [Light Ethereum Subprotocol](https://github.com/ethereum/devp2p/blob/master/caps/les.md): protocol used by light clients
+
+To debug and develop these networking components, Geth includes a command 
+line tool called `devp2p`.
+
+This page will outline some of `devp2p`s built-in tools.
+
+## ENR Decoding
+
+Ethereum Node Records can be decoded, verified and displayed to the terminal 
+using `enrdump`. It takes the ENR in its encoded form, which is the base64 
+encoding of its RLP representation. A decoded human-readable text representation 
+is displayed.
+
+Use `devp2p enrdump <base64>` to verify and display an Ethereum Node Record.
+
```

Comment:
Maybe an example? 
```
./devp2p enrdump  "enr:-J24QG3pjTFObcDvTOTJr2qPOTDH3-YxDqS47Ylm-kgM5BUwb1oD5Id6fSRTfUzTahTa7y4TWx_HSV7wri7T6iYtyAQHg2V0aMfGhLjGKZ2AgmlkgnY0gmlwhJ1a19CJc2VjcDI1NmsxoQPlCNb7N__vcnsNC8YYkFkmNj8mibnR5NuvSowcRZsLU4RzbmFwwIN0Y3CCdl-DdWRwgnZf"
Node ID: 001816492db22f7572e9eea1c871a2ffe75c28162a9fbc5a9d240e480a7c176f
URLv4:   enode://e508d6fb37ffef727b0d0bc618905926363f2689b9d1e4dbaf4a8c1c459b0b534dcdf84342b78250a6dc013c9ee9f89d095d7a6d1ef0c5f4c57a083b22c557ef@157.90.215.208:30303
Record has sequence number 7 and 7 key/value pairs.
  "eth"       c7c684b8c6299d80
  "id"        "v4"
  "ip"        157.90.215.208
  "secp256k1" a103e508d6fb37ffef727b0d0bc618905926363f2689b9d1e4dbaf4a8c1c459b0b53
  "snap"      c0
  "tcp"       30303
  "udp"       30303
```
Taken from one of the nodes here: https://github.com/ethereum/discv4-dns-lists/blob/master/snap.goerli.ethdisco.net/nodes.json